scene/gui/code_edit.h

```diff
@@ -57,6 +57,9 @@ class CodeEdit : public TextEdit {
 	int indent_size = 4;
 	String indent_text = "\t";
 
+	int line_height = 0;
+	int line_text_y_cache = 0;
```

Comment:
This should be under the line number section, line 112.

Reply:
removed from patch